frontend/debian/changelog

```diff
@@ -1,7 +1,17 @@
 cuttlefish-frontend (0.9.29) UNRELEASED; urgency=medium
 
-  * Make cuttlefish-orchestration depend on cuttlefish-user
-  * Make host orchestrator users configurable
+  [ cuttlefish-user ]
+  * Make host orchestrator users configurable.
+  * Support for `cvd stop`.
+  * User service account credentials if running on GCE.
+  * Use upstream `cvd`. 
+  * Extract ".zip" and ".tar.gz" user artifacts.
+  * Support `cvd powerwash`.
+
+  [ cuttlefish-orchestration ]
+  * Make cuttlefish-orchestration depend on cuttlefish-user.
+  * Start using nginx as a reverse proxy.
+  * Start using `systemd-journal-gatewayd` to expose host logs.
 
  -- Jorge Moreira <jemoreira@google.com>  Thu, 12 Oct 2023 18:12:35 -0700
```

Comment:
This line should also change to your name and today's date. You can use `dch` to make sure the correct format is used too.

Reply:
Done.